arangod/Aql/ExecutionBlockImpl.h

```diff
@@ -218,13 +218,20 @@ class ExecutionBlockImpl final : public ExecutionBlock {
 #endif
 
  private:
+  struct Context {
+    Context(ExecutionBlockImpl& block, AqlCallStack const& callstack);
```

Comment:
Perhaps we can change the name to something a little less generic? Perhaps `AqlCallContext` or so, or maybe you have another idea?

Reply:
This `Context` is only used when executing `executeWithoutTrace` and captures variables that are passed to functions that are called as part of `executeWithoutTrace`. That should now also make it easier to extract some parts of `executeWithoutTrace` and move them to separate functions.
So how about `ExecutionContext`?

Reply:
Sounds good!